rifServices/src/main/java/org/sahsu/rif/services/rest/WebService.java

```diff
@@ -1388,10 +1388,68 @@ private RIFStudySubmission getRIFSubmissionFromJSONSource(
 			JSONObject rifJobSubmission = jsonObject.optJSONObject("rif_job_submission");
 			if (rifJobSubmission != null) {
 				JSONObject studySelection = rifJobSubmission.optJSONObject("study_selection");
+				JSONObject disease_mapping_study = rifJobSubmission.optJSONObject("disease_mapping_study");
+				JSONObject risk_analysis_study = rifJobSubmission.optJSONObject("risk_analysis_study");
+				String name = "<no name>";
+				String description = "<no description>";
+				
+				if (disease_mapping_study != null) {
+					name = disease_mapping_study.optString("name");
+					description = disease_mapping_study.optString("description");	
+				}
+				if (risk_analysis_study != null) {
+					name = risk_analysis_study.optString("name");
+					description = risk_analysis_study.optString("description");
+				}
+				
+				String riskAnalysisDescription=null;
+				String studyType=null;
 				if (studySelection != null) {
-					rifLogger.info(this.getClass(), "ARWS - study_selection: " + studySelection.toString(2));	
+					riskAnalysisDescription = studySelection.optString("riskAnalysisDescription");	
+					studyType = studySelection.optString("studyType");	
+					if (riskAnalysisDescription != null && riskAnalysisDescription.length() > 0) {
+						if (studyType.equals("risk_analysis_study")) {
```

Comment:
Should do the null check first. Line 1411 could NPE if `studySelection.optString` returned null, above.